Sources/MongoSwift/MongoCollection.swift

```diff
@@ -84,11 +95,17 @@ public class MongoCollection<T: Codable> {
         self._collection = nil
     }
 
-    /// Drops this collection from its parent database.
-    /// - Throws:
-    ///   - `ServerError.commandError` if an error occurs that prevents the command from executing.
-    public func drop(session: ClientSession? = nil) throws {
-        let operation = DropCollectionOperation(collection: self, session: session)
+    /**
```

Comment:
```suggestion
   /**
```
super small nit: slightly misaligned